src/driver/mongodb/MongoDriver.ts

```diff
@@ -107,6 +107,12 @@ export class MongoDriver implements Driver {
      */
     dataTypeDefaults: DataTypeDefaults;
 
+    /**
+     * No documentation specifying a maximum length for identifiers could be found
+     * for MongoDB.
+     */
+    maxAliasLength: number;
```

Comment:
Maybe initialize this prop to -1 as infinity? I do not have any special reason. Just looking better :)

Reply:
I think this can lead to counterintuitive logics like `if (maxAliasLength !== -1)`, which looks weird in TypeScript, where we'd usually set the property as optional and consider it only if a value is associated. Don't you think so?

Reply:
I think this can lead to very counterintuitive logic like `if (maxAliasLength !== -1)` which is kinda "C-style checking"; I'd rather set the property `optional` and only consider its numeric value when it has one. What do you think?